commands/team_users_test.go

```diff
@@ -0,0 +1,372 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestArchiveUserCmd() {
```

Comment:
And their correspondant test functions:

```suggestion
func (s *MmctlUnitTestSuite) TestTeamUsersArchiveCmd() {
```

Reply:
Uops sorry